annotation-web/src/main/java/eu/europeana/annotation/web/service/impl/BaseAnnotationValidator.java

```diff
@@ -90,6 +91,7 @@ protected void validateAgentEntityBody(Body body) throws ParamValidationExceptio
     }
 
     protected void validateGeoTag(Body body) throws ParamValidationException {
+	if(body==null) return;
```

Comment:
if the argument is null, that is an error case that cannot be ignored. To avoid NPE, a PropertyValidationException must be thrown

Reply:
Improved.